src/Blzr.BootstrapSelect/BootstrapSelect.razor.cs

```diff
@@ -74,6 +78,13 @@ public partial class BootstrapSelect<TItem, TType> : ComponentBase
         [Parameter] public Expression<Func<TType>> ValueExpression { get; set; }
 
         [Parameter] public bool IsMultiple { get; set; }
+        
+        [Parameter] public bool UseIcon { get; set; }
```

Comment:
Do you need this and UseSubtext? Or can we can we make them private and calculate them based on the presence of a value in IconField or SubTexField? SO you only pass in the Field and the presence of the field assumes you want to use an Icon or Subtext?